tests/sentry/api/endpoints/test_event_attachment_details.py

```diff
@@ -138,6 +141,35 @@ def test_delete(self):
         assert response.status_code == 204, response.content
         assert EventAttachment.objects.count() == 0
 
+    @with_feature("organizations:event-attachments")
+    def test_delete_activity_no_group(self):
+        self.login_as(user=self.user)
+
+        self.create_attachment(group_id=None)
+        path = f"/api/0/projects/{self.organization.slug}/{self.project.slug}/events/{self.event.event_id}/attachments/{self.attachment.id}/"
+        response = self.client.delete(path)
+        assert response.status_code == 204
+
+        delete_activity = Activity.objects.get(type=ActivityType.DELETED_ATTACHMENT.value)
+        assert delete_activity.project == self.project
+        assert delete_activity.group_id is None
+        assert delete_activity.group is None
```

Comment:
what happens if there's no `group_id`? does it not associate with the issue/feedback?

Reply:
Good point - didn't think this through from the frontend perspective. It probably won't display, so maybe we shouldn't create the activity. Do all feedbacks have a group? @JoshFerge

Reply:
all feedbacks should be associated with an issue (group) -- the feedback `id` is basically the group/issue `id`

Reply:
Makes sense... and the attachment is created with the same group, right? I'll change it to only create an Activity when group is not None

Reply:
makes sense to me!